src/main/java/org/spongepowered/common/mixin/api/mcp/entity/monster/RavagerEntityMixin_API.java

```diff
@@ -0,0 +1,46 @@
+package org.spongepowered.common.mixin.api.mcp.entity.monster;
+
+import net.minecraft.entity.monster.RavagerEntity;
+import org.spongepowered.api.entity.living.monster.raider.Ravager;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(RavagerEntity.class)
+public abstract class RavagerEntityMixin_API extends AbstractRaiderEntityMixin_API implements Ravager {
+
+    @Shadow protected abstract boolean shadow$isMovementBlocked();
+
+    @Shadow private int stunTick;
+    @Shadow private int roarTick;
+
+    public boolean isImmobilized() {
```

Comment:
If this is implementing a method, please annotate appropriately with `@Override`